xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -1936,9 +1951,11 @@ bool CActiveAE::RunStages()
       if ((*it)->m_inputBuffers->m_format.m_dataFormat == AE_FMT_RAW)
       {
         bool isTrueHD = false;
+#if defined(TARGET_ANDROID)
+        //Need to ajdust duration for TrueHD as duration is now half (24 audio units to 12)
```

Comment:
Clear no. The Packing was changed for every platform. If this is needed for Android only, then something is wrong there.